youtube_dl/extractor/polskieradio.py

```diff
@@ -15,6 +16,63 @@
 )
 
 
+class PolskieRadioProgrammeIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?polskieradio\.pl/(?P<id>\d+/\d+)(,.*)?(\?.*)?$'
```

Comment:
So that makes two extractors interdependent with one another. This doesn't seem like a good practice to me when one extractor's ability to work with an URL is defined as a function of another extractor's suitability, versus a situation where each has a cleanly defined URL scope that it can process, as is the case now.

I'll make this change so that we can move forward but this is what I'm thinking.